examples/sycl/pvc/pvc_bfloat_dpas_gemm_cute.cpp

```diff
@@ -219,7 +221,7 @@ struct ExampleRunner {
     auto [M, N, K, L] = problem_shape_MNKL;
 
     stride_A = cutlass::make_cute_packed_stride(StrideA{}, cute::make_shape(M, K, L));
-    stride_B = cutlass::make_cute_packed_stride(StrideB{}, cute::make_shape(K, N, L));
+    stride_B = cutlass::make_cute_packed_stride(StrideB{}, cute::make_shape(N, K, L));
```

Comment:
Yes, the examples for Ampere in the other PRs all use `N != K`, and I just double checked it for PVC, and it is working as well. Basically, the shape is the matrix shape (row, column, batch), and the stride will define the leading dimension, so we don't need to worry about row/column major when passing the parameters.

> Also, we might want to change the way we create the shape for `copy` tensors here
> 
> https://github.com/codeplaysoftware/cutlass-fork/blob/ce10e06108b6074c7a2c785d03607c300bc72bdd/include/cutlass/gemm/collective/intel_pvc_mma.hpp#L153
> 
> too if we merge your change in the example.

Yes, we should pass (N, K, L) here too, similar to the SM90 implementation. I changed that but the implementation is failing. I'll look into it in a follow up PR

Reply:
I think the test failures are because the strides no longer match the shape once you pass `N, K, L` instead of `K, N, L` because of the way they have written the `TagtoStride` method used here: https://github.com/codeplaysoftware/cutlass-fork/blob/ce10e06108b6074c7a2c785d03607c300bc72bdd/examples/sycl/pvc/pvc_bfloat_dpas_gemm_cute.cpp#L381

Reply:
I think there is something else. For example, I think `vnni_matrix(b_vnni.data(), b.data(), L, K, N, 2)` is transposing B to create the VNNI matrix.

Reply:
I'll look into this in a follow-up PR as it will require a deeper investigation.